doc/config-schema.md

```diff
@@ -30,6 +30,7 @@ Each object in the `libraries` list represents a single library and has the foll
 | `id`           | string | A unique identifier for the library, in a language-specific format. It should not be empty and only contains alphanumeric characters, slashes, periods, underscores, and hyphens. | Yes      | Must be a valid library ID.                               |
 | `next_version` | string | The next released version of the library. Ignored unless it would increase the release version.                                                                                   | No       | Must be a valid semantic version, "v" prefix is optional. |
 | `generate_blocked` | bool | Set this to `true` to skip the generation of this library. It's `false` by default. | No       |  |
+| `release_blocked`  | bool   | Set this to `true` to skip the release of this library. It's `false` by default. | No       |  |
```

Comment:
Maybe we can note that this value is ignored if the `--version` flag is set by the user (`--version` flag has priority/ overrides)?

If this is already noted elsewhere that cli flags override the config.yaml file, then ignore this comment

Reply:
It's `--library` flag.

Reply:
Updated. PTAL.